src/controllers/handlers/scene-admin-handlers/add-scene-admin-handler.ts

```diff
@@ -36,10 +36,18 @@ export async function addSceneAdminHandler(
   const place = await getPlace(isWorlds, realmName, parcel)
 
   const isOwner = isWorlds
-    ? await hasWorldPermission(authenticatedAddress, place.world_name!)
+    ? await hasWorldOwnerPermission(authenticatedAddress, place.world_name!)
     : await hasLandPermission(authenticatedAddress, place.positions)
```

Comment:
What about moving this logic to the `scene-fetcher`? We are using it in multiple handlers.